README.md

```diff
@@ -103,6 +103,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [secp256k1.cr](https://github.com/q9f/secp256k1.cr) - Elliptic curve used in the public-private-key cryptography
 
 ## C bindings
+ * [augeas.cr](https://github.com/fernandes/augeas.cr) - Bindings for the [Augeas](https://augeas.net/)
```

Comment:
```suggestion
 * [augeas.cr](https://github.com/fernandes/augeas.cr) - Bindings for [Augeas](https://augeas.net/)
```

Reply:
it's a define article for *the* library, but ok, no practical difference on the description